lib/get-info.js

```diff
@@ -827,54 +828,40 @@ var Getters = {
             }
 
             try {
+                var $ = cheerio.load(res, { xmlMode: true });
+                switch ($("gphoto\\:videostatus").text()) {
+                    case "final":
+                        break; /* Incoming Fun. */
+                    case "pending":
+                        return cb("The video is still being processed.", null);
+                    case "failed":
+                        return cb("A processing error has occured and the video should be deleted.", null);
+                    /*
+                     *  This is the status where the video would seem completely ready and was playable on G+
+                     *    but would refuse to queue using the old technique because the duration wasn't populated.
+                     *  I'm not sure if that will still be the case using proper requests.
+                     *
+                     */
+                    //case "ready":
```

Comment:
Is it feasible to test whether it's necessary to handle this case?  Probably better safe than sorry in any case.